CoinApp/Views/CoinCell.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  CoinCell.swift
+//  CoinCapApiCall
+//
+//  Created by Nikita Shvad on 31.10.2021.
+//
+
+import UIKit
+
+class CoinCell: UITableViewCell {
+   
+    private lazy var stackView: UIStackView = {
+        let stackView = UIStackView.init(arrangedSubviews: [coinNameLabel,coinPriceLabel])
```

Comment:
```suggestion
        let stackView = UIStackView(arrangedSubviews: [coinNameLabel, coinPriceLabel])
```

Не нужно явно писать `.init`. Также не забываем про пробелы после запятой